tests/test_crypto.py

```diff
@@ -78,14 +78,79 @@ def test_gen_key_derivation(monero):
 
     assert expected == monero.xor_cipher(_d_in, b"\x55")  # decrypt _d_in
 
-def test_derive_view_tag(monero):
-    expected_view_tag: bytes = bytes.fromhex("76")
-
-    derivation: bytes = bytes.fromhex("0fc47054f355ced4d67de73bfa12e4c7"
-                                      "8ff19089548fffa7d07a674741860f97")
 
-    output_index = 0
+class Derivation_Test:
+    _derivation: bytes
+    _output_index: bytes
+    _expected_view_tag: int
+
+    def __init__(self, derivation, output_index, expected_view_tag):
+        self._derivation = bytes.fromhex(derivation)
+        self._output_index = (output_index).to_bytes(4, byteorder='big')
+        self._expected_view_tag = expected_view_tag
+
+    def do_test(self, monero):
+        assert self._expected_view_tag == monero.derive_view_tag(self._derivation, self._output_index)
```

Comment:
Suggestion 2: try changing `do_test` to:

```
 def do_test(self, monero):
        self._derivation = monero.xor_cipher(self._derivation, b"\x55") # encrypt with dummy key 0x55
        assert self._expected_view_tag == monero.derive_view_tag(self._derivation, self._output_index)
```

Reply:
Thanks for the suggestions, tests are working again, I'll send you the PR and if it's ok for you it will soon be deployed on our test provider